test/e2e/specs/various/navigation.spec.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Navigation block', () => {
+	test( 'When a menu item is active, it should have the .current-menu-item class name', async ( {
+		admin,
+		editor,
+		page,
+	} ) => {
+		const random = getRandomInt( 9999 );
+		const pageTitle = `Page ${ random }`;
+
+		await admin.createNewPost( {
+			postType: 'page',
+			title: pageTitle,
+		} );
+		await editor.publishPost();
```

Comment:
I think the `requestUtils` can be used to create a new page via the REST API here. The response would then contain the URL for the created page, so you wouldn't have to find it in the UI.

Reply:
Oh, the created page should also be deleted after the test using a `test.afterAll`.

It could be an option to improve the existing `deleteAllPosts` util to support a post type param, or alternatively copy its implementation but change 'page' to 'post'.